php/examples/identifiers.php

```diff
@@ -1,4 +1,10 @@
 <?php
 
-$привет_мир = 42;
-$𠮷 = "𠮷";
+// Comment these out until ASCII vs UTF-8 with large codepoints is
+// resolved.
+// See discussion at https://github.com/antlr/grammars-v4/pull/2576#issuecomment-1107839866
```

Comment:
It probably makes sense to point out the concrete issue as well.

Reply:
This file was changed so you can get your PR to work. It's only a placeholder. To make that clear, I'm adding the phrase "TODO" in a comment in this file.

There are two issues to resolve:
* The Maven Antlr testing plugin still uses ANTLRFileStream which does not support Unicode UTF-8. It only happens with "case insensitive" streams, but if you're going to rewrite grammars to use the new option, then it's not as important.
* The implementation supports Unicode variable names; the "spec" does not address the issue, and actually says otherwise. We need to decide whether to support Unicode names because people seemed to make use of this "feature". If so, the grammar will have to be changed.